content/manuals/dhi/how-to/policies.md

```diff
@@ -33,23 +40,19 @@ Docker Scout automatically evaluates policy compliance when new images are
 pushed. Each policy includes a compliance result and a link to the affected
 images and layers.
 
-## Create policies for your DHI-based images
-
-To ensure that the images you build using Docker Hardened Images remain secure,
-you can create Docker Scout policies tailored to your requirements for your own
-repositories. These policies help enforce security standards such as preventing
-high-severity vulnerabilities, requiring up-to-date base images, or validating
-the presence of key metadata.
+## Evaluate DHI policy compliance for your images
 
-Policies evaluate images when they are pushed to a repository, allowing you to
-track compliance, get notified of deviations, and integrate policy checks into
-your CI/CD pipeline.
+When you enable Docker Scout for your repositories, Docker Scout automatically
+provides default policies, including the [**Valid Docker Hardened Image (DHI) or
```

Comment:
The DHI policy is not a default one. It is an optional one that customers can configure if they wish.